string mode_switch_action
---
bool success
